README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+    - [DEMO LINK](https://tarasavage.github.io/js_task_parse_list_DOM/)
```

Comment:
It seems that you forgot to deploy your code. Deploy it and add the demo link to the description
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/211155365-d15fc65c-5e8e-4459-9e02-ab4bbc32949f.png">